tests/test_suites/GaneshaTests/test_nfs_ganesha_file_corruption_on_master_failover.sh

```diff
@@ -25,6 +25,11 @@ get_checksum() {
 	sha256sum $1 | awk '{ print $1 }'
 }
 
+# Function to get the checksum of a given file, using direct I/O
+get_checksum_with_direct_io() {
+	dd if="$1" iflag=direct bs=1M 2>/dev/null | sha256sum | awk '{ print $1 }'
```

Comment:
Consider checking if input variable has read access. See below message on function call.

Reply:
This check was done in the previous version of the test.
However, the new value of `io_retries` handles correctly the read access.
That's why it was removed.